lib/common.js

```diff
@@ -198,9 +198,11 @@ function stringifyRequest(options, body) {
 
   if (port) port = ':' + port;
 
+  var path = options.path ? options.path : '';
+
   var log = {
     method: method,
-    url: options.proto + '://' + options.hostname + port + options.path,
+    url: options.proto + '://' + options.hostname + port + path,
```

Comment:
previously, if path not specified and no match for request, it log:
`Nock: No match for request {   "method": "GET",   "url": "https://example.comundefined" }`